iml-gui/entries/templates/head_scripts.hbs

```diff
@@ -0,0 +1,2 @@
+{{! Because of Edge, see https://github.com/samthor/fast-text-encoding }}
+<script type="text/javascript" src="/static/text-polyfill.min.js"></script>
```

Comment:
Is this needed? We don't support IE still right?

Reply:
For edge actually. Probably not, but we can always remove it later.